components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/IdentityProviderManagementService.java

```diff
@@ -189,6 +189,24 @@ public void deleteIdP(String idPName) throws IdentityProviderManagementException
     }
 
     /**
+     * Forcefully deletes an Identity Provider from the logged-in tenant. This will remove all associations this
+     * Identity Provider had with any Service Provider within the tenant
+     *
+     * @param idPName Name of the IdP to be deleted
+     * @throws IdentityProviderManagementException Error when deleting Identity Provider
+     */
+    public void forceDeleteIdP(String idPName) throws IdentityProviderManagementException {
```

Comment:
Isn't this an axis2 service ? Doesn't this cause WSDL changes ? Did you add the changed wsdl ? Is that a separate commit ?.